src/components/TreeView/TreeView.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import TreeViewNode from './TreeViewNode';
+
+const TreeView = ({
+  nodes,
+  selectNode,
+  highlightOnSelect,
+  highlightOnHover
+}) => {
+  const classes = cx('list-group', {
+    'treeview-pf-select': highlightOnSelect,
+    'treeview-pf-hover': highlightOnHover
+  });
+
+  return (
+    <div className="treeview">
+      <ul className={classes}>
+        {nodes &&
+          nodes.map((node, index) => (
+            <TreeViewNode
+              node={node}
+              key={index}
+              level={1}
+              selectNode={selectNode}
+            />
+          ))}
+      </ul>
+    </div>
+  );
+};
+
+TreeView.propTypes = {
+  nodes: PropTypes.array,
+  selectNode: PropTypes.func,
+  highlightOnHover: PropTypes.bool,
+  highlightOnSelect: PropTypes.bool
```

Comment:
Comments for the storybook would be helpful here. What fields should the nodes have?

Reply:
There should be defaults for highlightOnHover and highlightOnSelect

Reply:
@jeff-phillips-18 how do you feel about me adding a story like this

<img width="1440" alt="fullscreen_2_7_18__12_20_pm" src="https://user-images.githubusercontent.com/15141412/35930969-64b7ba5a-0c01-11e8-8030-dc320e9a8355.png">

with regard to https://github.com/patternfly/patternfly-react/pull/196#discussion_r166654514?

Reply:
Is there a way to do it via comments had have it look decent in the props table?

Reply:
oohhh, ok, sorry @jeff-phillips-18, I'm picking up what you're putting down now.  Yes, I'll look into that and including the source for the other storybook example.  Thanks for pointing that out!